src/XdebugHandler.php

```diff
@@ -21,16 +21,20 @@ class XdebugHandler
     const RESTART_ID = 'internal';
 
     private static $skipped;
+    private $colorOption;
     private $loaded;
     private $envScanDir;
     private $version;
     private $tmpIni;
 
     /**
      * Constructor
+     *
+     * @param $colorOption string Command-line long option to force color output
      */
-    public function __construct()
+    public function __construct($colorOption = '')
```

Comment:
Perhaps an array, if multiple possibilities exist?

Reply:
There should only be one option to force color output, shouldn't there? In the `color=` format you might have `always/true`, `never/false` and `auto`, the first being the one to force color output.

Reply:
It was just a suggestion.

While I agree a single option is probably the most likely scenario, experience has taught me to always expect the unexpected :d

But I am perfectly happy with just assuming sane defaults for now.

Reply:
Yes, you are right. My thinking was to cover the common options and keep it simple. If a situation does not fit, then color output won't be returned. If it is required, then users will have to include the specific option on the command-line.

And we can always improve it later.

Reply:
@alcohol Are you happy for me to merge this?